CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## [v2.11.9](https://github.com/liferay/alloy-editor/tree/v2.11.9) (2020-07-02)
+
+[Full changelog](https://github.com/liferay/alloy-editor/compare/v2.11.7...v2.11.9)
+
+### :new: Features
+
+-   feat: make the toolbar visible in tables ([\#1404](https://github.com/liferay/alloy-editor/pull/1404))
+
+### :wrench: Bug fixes
+
+-   fix: update instructions at end of build script ([\#1405](https://github.com/liferay/alloy-editor/pull/1405))
+
+### :house: Chores
+
+-   chore: update CKEditor build to v4.14.1 ([\#1402](https://github.com/liferay/alloy-editor/pull/1402))
```

Comment:
Given that these are repeat of what is in v2.11.8, I think we have two choices:

1. Edit the changelog by hand to show what really happened; or two:
2. Send a PR updating the submodule, so that we'll get a correct changelog "for free".

I think "2" is probably the more correct one, so sent this PR for that: https://github.com/liferay/alloy-editor/pull/1407